index.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module "asyncify-wasm" {
+  class Instance extends WebAssembly.Instance {
+    constructor(module: WebAssembly.Module, imports: WebAssembly.Imports);
+  }
+
+  function instantiate(
```

Comment:
Have you seen my comment on the issue? Let's define those in terms of `const ...` so that they definitely stay in sync with upstream API.

Reply:
This is much better than redeclaring, thanks for the suggestion!